### ISSUE TITLE
Needing help understanding what is written.

### ISSUE THREAD

**Issue by Just-another-pleb** [Paul-Reed/weather-icons#5]
Paul,

I have installed the icons a long time ago.

Node-Red is a lot different now.

This bit is confusing me:

> and also if you havent already done so, httpAdminRoot must also be used to make the editor UI available at a path other than /. So for example if you changed the setting to httpAdminRoot: '/admin', then instead of your editor being accessed at the URL 192.168.168.13:1880/ you would use 192.168.168.13:1880/admin

If I do that and try to run NR, I get an error on the screen something about can't open `/`

(Sorry, also going back to the "public" path.  Does it end with a `/` or not?)
On one machine is it:

`    httpStatic: 'home/pi/.node-red/public',`

But on another it is:
`    httpStatic: 'home/pi/.node-red/public/',`

Both work.
Yes, I know that is the same one pasted twice.   Just to show the question clearer.

I have also set the theme to use angular ..... thingy.

I know what I am missing will be simple.
Though I'd appreciate being reminded.

BTW, also:  (just asking)   Are those docs up to speed with NR 1.1.3?
I've been caught with other new things in the `settings.js` file on other things.

Thanks

**Comment by Paul-Reed** [comment]
If you check my readme, you will see "**DO NOT USE - SEE WEATHER-ICONS-LITE WHICH IS MERGED INTO NODE-RED CORE**"

The reason for me saying DO NOT USE is that the project has now been merged into the node-RED dashboard, so if you have the NR dashboard installed, then the weather icons will already be installed by default and ready for you to use.
Trying to install them again will result in problems!
If you want to know how to use the weather icons in node-RED, take a look at https://github.com/Paul-Reed/weather-icons-lite/blob/master/README.md

**Comment by Just-another-pleb** [comment]
(If you read this)

It isn't that simple.

On the new machine I have Node-Red 1.1.3.

I imported a flow from another machine which used the weather icons - unknowingly.  (I'll get to that soon)

So as is:  The flow works on the other machine.   It has the weather icons installed because it was done years ago.

The new machine:  not.   I only exported the imported the nodes.

I deployed and.....   It did not work.

So that is against what you said.
I am using the sunrise and sunset icons.

So being confused to why, I looked, and then realised I needed the weather icons.
I installed them.

So though you say they **are** included in the latest NR (and dashboard which is also up to date) they didn't work when I imported the flow.
